src/main/java/tech/gruppone/stalker/server/model/api/UserDataDto.java

```diff
@@ -2,12 +2,16 @@
 
 import java.sql.Timestamp;
 import java.time.LocalDate;
+import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 import lombok.NonNull;
 
-@Builder
 @Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
```suggestion
@Data
@Value
```

dovrebbe essere sufficiente questo